src/main/webapp/data/userMapData.json

```diff
@@ -1661,6 +1712,7 @@
       "The University of Tampa",
       "Thiel College",
       "Thomas Ewing Junior High School",
+      "Thomas Jefferson University",
```

Comment:
This should be one line below to follow alphabetical order.

Reply:
@whipermr5 All done)